styles/global.scss

```diff
@@ -435,7 +435,7 @@ a.has-text-grey {
 
 .k-text-grey {
   @include ktheme() {
-    background-color: theme('k-grey');
+    color: theme('k-grey');
```

Comment:
doesn't this break something cc @roiLeo @daiagi

Reply:
It's outdated. I have changed `k-text-grey` to `has-text-k-grey`